jsignpdf/src/main/resources/net/sf/jsignpdf/translations/messages.properties

```diff
@@ -145,102 +145,102 @@ gui.vs.acro6layers.checkbox=Acrobat 6 layer mode
-hlp.location=location of a signatue (e.g. Washington DC). Empty by default.
+hlp.encryption=Encryption mode for the output PDF default value is NONE. Possible values are {0}. Use togethter with -upwd and -opwd in case of PASSWORD mode, and -ec in case of CERTIFICATE
+hlp.footer.examples=Examples:\r\n---------\r\n$ java -jar JSignPDF.jar -kst WINDOWS-MY mydocument.pdf\r\n-> creates copy of mydocument.pdf with name mydocument_signed.pdf, which is digitally signed with the first certificate found in default windows certificate store\r\n------------\r\n$ java -jar JSignPDF.jar -kst PKCS12 -ksf my_certificate.pfx -ksp myPrivateKeystorePassword -ka cert23 -pe PASSWORD -opwd xxx123 -upwd 123xxx -pr DISALLOW_PRINTING mydocument.pdf\r\n-> creates signed and encrypted file mydocument_signed.pdf, printing of the new file is not allowed. For signature is used key with alias cert23 from the file my_certificate.pfx\r\n------------\r\n$ java -jar JSignPDF.jar -lkt\r\n-> lists keystore types\r\n------------\r\n$ java -jar JSignPDF.jar -kst PKCS12 -ksf my_certificate.pfx -ksp myVeryPrivatePassword -lk -q\r\n-> list names (aliases) of keys stored in my_certificate.pfx file using the password for keystore. Quiet mode is enabled so no debug info is printed.
+hlp.footer.exitCodes=Program exit codes:\r\n-------------------\r\n0 program finished without error\r\n1 command line is in a wrong format\r\n  2 no operation requested - e.g. no file for signing provided\r\n  3 signing of some - but not all - files failed\r\n  4 signing of all files failed
+hlp.hashAlgorithm=Hash algorithm used for signature. The default value is SHA-1. Available values are {0}.
+hlp.header=JSignPDF is designed to digitally sign PDF documents. If you start the program without any commandline argument, the GUI is started, otherwise you can use it in commandline batch mode.
+hlp.help=Prints this help screen
+hlp.imgPath=Image path for visible signature
+hlp.keyAlias=Name (alias) of the key, which should be used for signing the document. If not given, the first key in the keystore is used. (List the key aliases using -lk)
+hlp.keyIndex=Zero-based index of the key, which should be used for signing the document. If neither this option nor alias is given, the first key (index=0) in the keystore is used. (List the key aliases using -lk). This option has lower priority than alias.
+hlp.keyPwd=Password for the key in the keystore. In most cases you don't need to set this option — only keystore is protected by a password, but just in case :)
+hlp.ksFile=Sets (JKS) keystore file — as the value use the path on which is file with private key(s) located (.p12, .pfx, .jks, …). Some keystores haven't keys stored in a file (e.g. Windows keystore — WINDOWS-MY), then don't use this option.
+hlp.ksPwd=Password to (JKS) keystore
+hlp.ksType=Sets (JKS) keystore type (you can list possible values for this option -lkt argument)
```

Comment:
Also here the `JKS` shouldn't be mentioned, as there are other types too.
And as help messages are printed to a terminal, I suggest avoiding the `…` character.